### DISCUSSION TITLE
悄悄发个告示 | /cn/2006/05/14-24-00/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1590] [Comments]
新的网站“统计之都”（Capital of Statistics）经过谢某与若干同伙的一阵密谋策划，已经开始逐步投入使用了。不过现在显得乱糟糟的，所以也不要太声张……嘘……
[&hellip;] 地址：https://cosx.org
[&hellip;] 谢某重出江湖之日，也不会远了。

https://yihui.org/cn/2006/05/14-24-00/

**Comment by hongtaoh** [comment]
它已经 18 周岁了。成年了。
